app/ui/legacy/src/main/java/com/fsck/k9/ui/account/AccountsViewModel.kt

```diff
@@ -1,8 +1,118 @@
 package com.fsck.k9.ui.account
 
+import android.content.ContentResolver
+import android.database.ContentObserver
+import android.os.Handler
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.fsck.k9.Account
+import com.fsck.k9.AccountsChangeListener
 import com.fsck.k9.Preferences
+import com.fsck.k9.controller.UnreadMessageCountProvider
+import com.fsck.k9.provider.EmailProvider
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.channels.awaitClose
+import kotlinx.coroutines.channels.sendBlocking
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.callbackFlow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.flatMapLatest
+import kotlinx.coroutines.launch
+import timber.log.Timber
 
-class AccountsViewModel(preferences: Preferences) : ViewModel() {
-    val accountsLiveData = AccountsLiveData(preferences)
+@ExperimentalCoroutinesApi
```

Comment:
This just kicks the warning down the line, i.e. users of `AccountsViewModel` will have to opt in to the experimental API. 
Since using `Flow` is an implementation detail of this class, please use the `@OptIn` annotation here.